specification/ai/data-plane/Face/stable/v1.2/Face.json

```diff
@@ -475,22 +475,65 @@
         ],
         "parameters": [
           {
-            "$ref": "#/parameters/CreateLivenessSessionContent.livenessOperationMode"
-          },
-          {
-            "$ref": "#/parameters/CreateLivenessSessionContent.deviceCorrelationIdSetInClient"
-          },
-          {
-            "$ref": "#/parameters/CreateLivenessSessionContent.enableSessionImage"
+            "name": "livenessOperationMode",
```

Comment:
here the parameters were moved to be inline, it doesn't really matter and I don't think its worth to cheat those back in with some spread as the definition name at least would need to change